File: eFormAPI/eFormAPI.Common/Models/Cases/Response/CaseListModel.cs
using System.Collections.Generic;
using eFormShared;

namespace eFormAPI.Common.Models.Cases.Response
{
    public class CaseListModel
    {
        public int NumOfElements { get; set; }
        public int PageNum { get; set; }
        public List<Case> Cases { get; set; }
    }
}

File: eform-client/src/app/common/models/cases/response/case-list.model.ts
import {CaseModel} from '../case.model';

export class CaseListModel {
  numOfElements: number;
  pageNum: number;
  cases: Array<CaseModel> = [];

  constructor() {
    this.pageNum = 0;
    this.numOfElements = 0;
  }
}
